### DISCUSSION TITLE
How do I run the JQ plaground locally?

### DISCUSSION THREAD

**Discussion by iainelder** [sandbox-bio/sandbox.bio#7] [Q&A] [answered]
Thanks for the awesome JQ playground tool.

I understand it evolved from an earlier tool called [jqkungfu](https://github.com/robertaboukhalil/jqkungfu).

jqkungfu allows me to run it locally. I like this feature because I feel safer about using it to compose queries with confidential data.

How do I do the same with sanbox-bio's version?

If only the hosted version is supported, where can I check which data is shared with the hosted version, where my data is stored, and how it is processed?

**Comment by robertaboukhalil** [comment]
Ah, hadn't thought about that use case, I will check how easy it would be to update jqkungfu to add a resizable input.

**Comment by iainelder** [comment]
Thanks for looking into it. An expanding textfield in jqkungfu would be helpful for complex queries.

I probably don't really want to run sandbox-bio locally since it has so many other parts that I'm not interested in just now.

**Reply by iainelder** [reply]
See original request in [jqkungfu issue sandbox-bio/feedback#7](https://github.com/robertaboukhalil/jqkungfu/issues/7).

**Comment by robertaboukhalil** [comment, marked as answer]
Added a resizable textarea to jqkungfu: https://github.com/robertaboukhalil/jqkungfu/issues/7
